src/benchmarks/micro/libraries/System.IO.Compression/ZipFileTests.cs

```diff
@@ -0,0 +1,42 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.IO.Compression;
+
+[BenchmarkCategory(Categories.Libraries, Categories.NoWASM)]
+public class ZipFileTests
+{
+    private static readonly string _testOutputDirectory = FileUtils.GetTestFilePath();
+    private static readonly string _testOutputZipFile = Path.Combine(_testOutputDirectory, "TestOutputZipFile.zip");
+    private static readonly string _testInputDirectory = Path.Combine(
+        AppContext.BaseDirectory, "libraries", "System.IO.Compression", "TestData");
+    private static readonly string _testInputZipFile = Path.Combine(
+        AppContext.BaseDirectory, "libraries", "System.IO.Compression", "TestInputZipFile.zip");
+
+    [IterationSetup(Target = nameof(ZipFileCreateFromDirectory))]
```

Comment:
How long does it take to run this benchmark? We usually avoid `[IterationSetup]` for the reasons described here:

https://github.com/dotnet/performance/blob/main/docs/microbenchmark-design-guidelines.md#iterationsetup

Does `ZipFile.CreateFromDirectory` overwrite the file?